atm.js

```diff
@@ -1,21 +1,123 @@
+      checkingAccnt.totalBal = 0;
+      checkingBox.val('');
+      checkBalanceDisplay.html('$' + checkingAccnt.totalBal);
+      savingsAccnt.withdrawal(Math.abs(checkingLeftOver));
+      if(savingsAccnt.totalBal === 0) {
+        savingsDisplay.addClass('zero');
+      }
+      savingsDisplay.html('$' + savingsAccnt.totalBal);
+      console.log('total checking balance:' + checkingAccnt.totalBal);
+  } else if (withdraw_amnt <= checkingAccnt.totalBal) {
+      checkingAccnt.withdrawal(withdraw_amnt);
+      checkingBox.val('');
+      if(checkingAccnt.totalBal === 0) {
+        checkBalanceDisplay.addClass('zero');
+      }
+      checkBalanceDisplay.html('$' + checkingAccnt.totalBal);
+      console.log('total savings balance:' + savingsAccnt.totalBal);
+      console.log('total checking balance:' + checkingAccnt.totalBal);
+  }
+  // If the withdrawal amount is more than the combined account balance, ignore it.
+  else {
+    console.log('overdraft warning! entered amount can\'t be converd');
+  }
+})
+  //savings account
+    //on click, deposit
+  savingsDBtn.click(function() {
+    savingsAccnt.deposit(parseInt(savingsBox.val()));
+    savingsBox.val('');
+    savingsDisplay.removeClass('zero');
+    savingsDisplay.html('$' + savingsAccnt.totalBal);
+    console.log('total savings balance:' + savingsAccnt.totalBal);
+  })
+    //on click, withdraw
+  savingsWBtn.click(function() {
 
-}
+    var withdraw_amnt = parseInt(savingsBox.val());
 
-function savingsWithdrawal(amount){
+    if (withdraw_amnt >= savingsAccnt.totalBal && savingsAccnt.totalBal >= 0 && withdraw_amnt <= checkingAccnt.totalBal) {
+      savingsDisplay.addClass('zero');
+      var savingsLeftOver = savingsAccnt.withdrawal(withdraw_amnt);
+      savingsAccnt.totalBal = 0;
+      savingsBox.val('');
+      savingsDisplay.html('$' + savingsAccnt.totalBal);
+      checkingAccnt.withdrawal(Math.abs(savingsLeftOver));
+      if(checkingAccnt.totalBal === 0) {
+        checkBalanceDisplay.addClass('zero');
+      }
+      checkBalanceDisplay.html('$' + checkingAccnt.totalBal);
+      console.log('total savings balance:' + savingsAccnt.totalBal);
+      console.log('total checking balance:' + checkingAccnt.totalBal);
 
-}
+    } else if (withdraw_amnt <= savingsAccnt.totalBal) {
+      savingsAccnt.withdrawal(withdraw_amnt);
+      savingsBox.val('');
+      if(savingsAccnt.totalBal === 0) {
+        savingsDisplay.addClass('zero');
+      }
+      savingsDisplay.html('$' + savingsAccnt.totalBal);
+      console.log('total savings balance:' + savingsAccnt.totalBal);
+    } else {
+      console.log('overdraft warning! entered amount can\t be converd');
+    }
+  })
 
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+
+
+
+// As a user, I want the color of my back account to reflect it's balance (there's a CSS class called .zero already written for this!)
+// Are there ways to refactor your code to make it DRYer or more Object-Oriented?
```

Comment:
Definitely! I think getting rid of the global variables would help as well as refactoring the callback functions for your click events (Please see my notes above)